etc/docker/Dockerfile.triton-jetpack-focal

```diff
@@ -77,7 +77,8 @@ RUN echo 'deb https://apt.kitware.com/ubuntu/ focal main' > /etc/apt/sources.lis
 #
 RUN apt-get -y update && apt-get -y --no-install-recommends install \
     build-essential \
-    cmake \
+    cmake-data=3.31.7-0kitware1ubuntu20.04.1 \
```

Comment:
Otherwise, we get CMake 4 which causes difficulties that I don't want to spend time resolving right now.